lib/diagnostic.js

```diff
@@ -7,27 +7,52 @@
 // (uppercase strings without punctuation)
 // and returns that array.
 const normalizeWords = function () {
-
+  let words = text.split(/\s+/);
+  let normalizedWords = [];
+  for (let i = 0; i < words.length; i++) {
+    let word = words[i];
+    normalizedWords[i] = word.replace(/\W+/, '').toUpperCase();
+  }
+  return normalizedWords;
 };
 
+
+
 // 2. Write a function that takes a string arguments
 // and returns an array of unique normalized words.
 const uniqueWords = function () {
+  let words = getNormalizedWords(text);
```

Comment:
getNormalizedWords won't work here because it was never defined. You should be calling ```normalizeWords(text)```